src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -199,6 +206,36 @@ def get_condition_query_groups(
     return condition_groups
 
 
+def get_condition_group_results(
+    queries_to_groups: dict[UniqueConditionQuery, set[int]]
+) -> dict[UniqueConditionQuery, dict[int, int]]:
+    condition_group_results = {}
+    current_time = timezone.now()
+
+    for unique_condition, group_ids in queries_to_groups.items():
+        handler = unique_condition.handler
+
+        _, duration = handler.intervals[unique_condition.interval]
+
+        comparison_interval: timedelta | None = None
+        if unique_condition.comparison_interval is not None:
+            comparison_interval = COMPARISON_INTERVALS_VALUES.get(
+                unique_condition.comparison_interval
+            )
+
+        result = safe_execute(
+            handler().get_rate_bulk,
```

Comment:
unfortunately i need to instantiate the condition handler to call `get_rate_bulk`, which calls `self.batch_query` which is defined differently depending on the type of slow condition (`EventFrequencyCondition`, `EventUniqueUserFrequencyCondition`, etc)